### DISCUSSION TITLE
About the bug of tool sql-pretty

### DISCUSSION THREAD

**Discussion by ZJesse** [he3-app/he3-feedback#294] [Bug Report]
### Version
sql-pretty 1.0.14

### Environment
Windows v1.3.23

### Steps to reproduce
格式化选项呢？

### What is expected?
![image](https://github.com/he3-app/he3-feedback/assets/3295680/1949047a-dae3-44fb-92b7-415b51afbcb8)

### What is actually happening?
没有选项可设置

**Comment by lyzhang1999** [comment]
感谢反馈，已创建 issues：https://github.com/he3-app/he3-feedback/issues/296
